helmfile/values/harbor.yaml.gotmpl

```diff
@@ -19,15 +19,15 @@ persistence:
   resourcePolicy: "keep"
   persistentVolumeClaim:
     chartmuseum:
-      size: 5Gi
+      size: {{ .Values.harbor.persistence.persistentVolumeClaim.chartmuseum.size }}
     registry:
-      size: 5Gi
+      size: {{ .Values.harbor.persistence.persistentVolumeClaim.registry.size }}
     jobservice:
-      size: 1Gi
+      size: {{ .Values.harbor.persistence.persistentVolumeClaim.jobservice.size }}
     database:
-      size: 1Gi
+      size: {{ .Values.harbor.persistence.persistentVolumeClaim.database.size }}
     redis:
-      size: 1Gi
+      size: {{ .Values.harbor.persistence.persistentVolumeClaim.redis.size }}
```

Comment:
These needs to be updated as well to match the new value structure

Reply:
I updated them but apparently I didn't commit that. Hmm, sorry for that, fixing right now.